docs/catalog/sinks/jira.rst

```diff
@@ -1,39 +1,34 @@
 Jira
 #################
 
-Robusta can send playbook results to Jira.
-
-To configure the Jira sink we will need some parameters to be set.
-
-.. note::
-
-    2-way interactivity (``CallbackBlock``) isn't implemented yet.
+Robusta can open Jira tickets based on playbooks results.
 
 Get your Jira configurations
 ------------------------------------------------
 
-To configure the Jira sink you need to have following values: ``url``, ``username``, ``api_key``,  ``dedups`` ``and project_name``.
-
-The ``url`` parameter is pretty simple to find: you need to copy the url of your workspace, usually
-it looks like https://workspace.atlassian.net (**Note:** schema (https) is required)
-
-The ``username`` it's the username you are logging with to Jira workspace.
+To configure the Jira sink you need to have following:
 
-``api_key`` can be configured, following the `instruction <https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/>`_.
+* ``url`` : The url of your workspace. For example: https://workspace.atlassian.net (**Note:** schema (https) is required)
+* ``username`` : Jira workspace user name. For example: jira-user@company.com
+* ``api_key`` : follow the `instructions <https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/>`_ to get your api key.
+* ``project_name`` : Project for the Jira tickets.
+* ``issue_type`` : [Optional - default: ``Task``] Jira ticket type
+* ``dedups`` : [Optional - default: ``fingerprint``] Tickets deduplication parameter. By default, Only one issue per ``fingerprint`` will be created. There can be more than one value to use. Possible values are: fingerprint, cluster_name, title, node, type, source, namespace, creation_date etc
+* ``project_type_id_override`` : [Optional - default: None] If available, will override the ``project_name`` configuration
+* ``issue_type_id_override`` : [Optional - default: None] If available, will override the ``issue_type`` configuration
```

Comment:
Its a bit weird to have issue_type_id_override override issue_type when the user can just set issue_type so I am assuming it was requesting to have a way to set the type by id number?

Reply:
The code use the `issue_type` to find the `issue_type_id`
Project name has a similar logic.

It's already happened to few users, that either the project or issue type resolution didn't work.
I added the overrides, to allow the user to just provide the IDs he wants to use.
Without it, you can use the Jira integration

Reply:
got it, thanks